---
float32 stop_result
